File: angular/src/app/models/journal/journalitems/journal-handout.model.ts
import { JournalItem } from './journal-item.model';
import { JournalItemType } from './journal-item-type.enum';

export class JournalHandout extends JournalItem {

    description?: string;
    ownerNotes?: string;

    constructor() {
        super(JournalItemType.Handout);

    }
}

File: dndapi-onion/Domain/RequestModels/Journal/JournalItems/JournalHandoutModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Domain.RequestModels.Journal.JournalItems
{
    public class JournalHandoutModel : JournalItemModel
    {
        public string Description { get; set; }
        public string OwnerNotes { get; set; }
        public JournalHandoutModel()
        {
            Type = Domain.JournalItems.JournalItemType.Handout;
        }
    }
}
